sdk/src/main/java/com/google/cloud/dataflow/sdk/io/PubsubIO.java

```diff
@@ -666,8 +754,11 @@ private Bound(String name, PubsubSubscription subscription, PubsubTopic topic,
 
       @Override
       public void populateDisplayData(DisplayData.Builder builder) {
-        super.populateDisplayData(builder);
-        populateCommonDisplayData(builder, timestampLabel, idLabel, topic);
+        String topicString =
+            topic == null ? null
+            : topic.isAccessible() ? topic.get().asPath()
+            : topic.toString();
+        populateCommonDisplayData(builder, timestampLabel, idLabel, topicString);
```

Comment:
Shouldn't we pass the VP into `populateCommonDisplayData` and avoid the logic here?